README.md

```diff
@@ -25,18 +25,16 @@
 ## Introduction
 
 The SDS SDK contains artifacts and resources for all parties that intent to use, extend or integrate with the BAMM
-Aspect Meta Model, e.g. Solution Developers, Domain Experts or OEMs.
+Aspect Meta Model, e.g., Solution Developers, Domain Experts or OEMs.
 
-At its core are components, which help working with the BAMM Aspect Meta Model (BAMM). The SDK comprises components to
-load and validate models such as the one depicted below and generate artifacts such as DTOs, static classes,
-documentation and so on.
-![BAMM Aspect Meta Model (BAMM) Elements](documentation/developer-guide/modules/getting-started-guide/images/solution-with-movement-aspect.svg)
+At its core are components, which help to work with the BAMM Aspect Meta Model (BAMM). The SDK comprises components to
+load and validate models generate artifacts such as static classes, documentation, DTO (data transfer object) and so on.
 
 To fully leverage BAMM, provided components include language-specific meta model implementations, code generators,
 validators etc.
 
-This document provides an overall overview of the SDK and the concepts applied throughout it. Detailed documentation and
-concepts for each component can be found in the respective subfolders.
+This document provides an overall overview of the SDK, and the concepts applied throughout it. Detailed documentation and
+concepts for each component can be found in the respective subfolder.
```

Comment:
This refers to the subfolders in which the different modules reside, so this should probably stay plural.

Reply:
My thought (and probably the one of the IDE highlighting the plural as an error) was that each component has its individual respective subfolder. At the same time, the subfolder may also contain subfolders, so I will change it back to plural.